scripts/apps/ingest/views/settings/ingest-routing-content.html

```diff
@@ -121,12 +121,12 @@ <h3 class="modal__heading" translate ng-hide="editScheme._id" translate>Add New
                 </div>
             </div>
             <div class="modal__body-footer">
-                <div class="modal_footer_btn">
-                    <div class="cancelBtn" ng-if="rule">
-                        <button class="btn" ng-click="cancelRule(rule)" translate>Cancel</button>
-                        <button class="btn btn--primary" ng-click="save()" ng-disabled="!rule.name" translate>Save</button>
-                    </div>
-                    <button class="btn btn--primary" ng-click="done()" ng-disabled="rule" translate>Done</button>
+                <div class="button-group button-group--end button-group--comfort" role="group">
+                    <span ng-if="rule">
```

Comment:
An inline element (span in this case),  should not be used as a container for block level elements (button, div, etc.). Use rather a div instead. Also, you can use a helper class on that element to avoid flex problems and the need for additional CSS, like so:
`<div class="sd-display--contents" ng-if="rule">`